centaur/src/main/resources/standardTestCases/draft3_same_named_inputs_priority.test

```diff
@@ -0,0 +1,17 @@
+name: draft3_same_named_inputs_priority
+testFormat: workflowsuccess
+backends: [Local]
+tags: [localdockertest]
```

Comment:
Since this is really a WOMification test you could probably tie this in to the `WdlFileToWomSpec` rather than resorting to centaur/local-only.

That would also allow us to have `draft-2`, `1.0` and `biscayne` editions without having to make separate centaur tests for each version